src/window.cpp

```diff
@@ -3061,38 +3061,44 @@ void InputLoop()
 	HandleAutoscroll();
 }
 
+static const uint TICK_15_INTERVAL = 15 * MILLISECONDS_PER_TICK;
+static const uint TICK_100_INTERVAL = 100 * MILLISECONDS_PER_TICK;
+
 /**
  * Update the continuously changing contents of the windows, such as the viewports
  */
 void UpdateWindows()
 {
 	Window *w;
 
-	static int highlight_timer = 1;
-	if (--highlight_timer == 0) {
-		highlight_timer = 15;
+	static uint32 next_highlight_time = _realtime_tick + TICK_15_INTERVAL;
+	if (_realtime_tick >= next_highlight_time)
+	{
+		next_highlight_time = _realtime_tick + TICK_15_INTERVAL;
```

Comment:
Awh, so in FF we still have the annoying color flickering like a mofo? :D

Reply:
One of the annoying things about FastForward, is that window highlights go like crazy, which feels really weird .. you expect the game to go faster, not the window stuff .. so I was wondering if I was reading this correctly that this meant on FF (FastForward), window highlighting is still going like crazy?

Reply:
I was under the impression that _realtime_tick is not affected by fast forward? I wen't by the suggestion https://github.com/OpenTTD/OpenTTD/pull/6744#issuecomment-383404260

Reply:
Ah :D That would explain ;)